src/components/NavBar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import CategoryFilter from './CategoryFilter';
+import '../stylesheets/NavBar.css';
+import { CHANGE_FILTER, CHANGE_VISIBILITY } from '../actions';
+
+const NavBar = ({ handleChangeFilter: changeFilter, handleVisibility: changeVisibity }) => {
+  const handleFilter = e => {
+    changeFilter(e.target.value);
+    changeVisibity(e.target.value);
+  };
+  return (
+    <nav className="d-flex flex-column flex-md-row">
+      <div className="nav-links d-flex flex-column flex-md-row">
+        <h1 className="title"><Link to="/">BookStore CMS</Link></h1>
+        <h2 className="books"><Link to="/books">BOOKS</Link></h2>
+        <h2 className="categories"><CategoryFilter handleFilter={handleFilter} /></h2>
+      </div>
+      <span className="login"><Link to="/login" className="fa fa-user-circle" /></span>
+    </nav>
+  );
+};
```

Comment:
- [ ] Your navbar needs to look exactly as the given one in the Zeplin webpage:
![image](https://user-images.githubusercontent.com/38522578/133334233-c8e9381d-fa1e-493c-8b5e-9171c29de8b5.png)

Yours:
![image](https://user-images.githubusercontent.com/38522578/133334376-cf67b1f9-1c84-426c-9caa-6819bd7a75a8.png)

- [ ] Also, some of the things that I marked are unaligned properly, please do so.
- [ ] There is more space in your right part of the page than in the left one, please center everything in order to comply with the project requirements.